src/Three.js

```diff
@@ -120,6 +120,7 @@ export { Sphere } from './math/Sphere.js';
 export { Ray } from './math/Ray.js';
 export { Matrix4 } from './math/Matrix4.js';
 export { Matrix3 } from './math/Matrix3.js';
+export { Matrix2 } from './math/Matrix2.js';
```

Comment:
Do you mind also adding the entry for `Three.WebGPU.js`? 

Otherwise the WebGPU build won't have this class.

Reply:
Done!

Reply:
Also docs. 🙏